bcipy/helpers/triggers.py

```diff
@@ -638,3 +639,185 @@ def read_triggers(triggers_file: TextIO) -> List[Tuple[str, str, float]]:
     for _, (name, trg_type, stamp) in enumerate(records):
         corrected.append((name, trg_type, float(stamp) + offset))
     return corrected
+
+
+class TriggerType(Enum):
+    """
+    Enum for the primary types of Triggers.
+    """
+
+    NONTARGET = "nontarget"
+    TARGET = "target"
+    FIXATION = "fixation"
+    PROMPT = "first_pres_target"
+
+    @classmethod
+    def list(cls):
+        """Returns all enum values as a list"""
+        return list(map(lambda c: c.value, cls))
+
+    def __str__(self):
+        return f'{self.value}'
+
+
+class Trigger:
+    """
+    Object that encompasses data for a single trigger instance.
+    """
+
+    def __init__(self,
+                 label: str,
+                 type: TriggerType,
+                 time: str):
+        self.label = label
+        self.type = type
+        self.time = time
+
+    def __repr__(self):
+        return f'Trigger: label=[{self.label}] type=[{self.type}] time=[{self.time}]'
+
+
+class FlushSensitivity(Enum):
```

Comment:
It seems like sensitivity refers to a relative level. Maybe a better term would be `FlushPeriod`, which indicates the interval of time between occurrences.

Reply:
Perhaps a rename to `FlushFrequency` might also work?